src/App.jsx

```diff
@@ -1,16 +1,18 @@
 import React from 'react';
 import './App.scss';
 
+import { Sum } from './components/Sum';
+
 const App = () => (
-  <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove comment lines :) */}
-  </>
+  <div className="container">
+    <div className="calculation">
+      <Sum a={2} b={5} />
+      <Sum a={-5} b={5} />
+      <Sum a={10} b={0} />
+      <Sum a={0} b={5} />
+      <Sum a={0} b={0} />
```

Comment:
In these 3 components you should pass params like that:
just a = 10
just b = 5
no params at all

Reply:
Oh, jeez, you're right!